src/psyclone/dynamo0p3.py

```diff
@@ -2994,28 +2990,22 @@ def _stub_declarations(self, parent):
     def _invoke_declarations(self, parent):
         '''
         Declare all LMA-related quantities in a PSy-layer routine.
+        Note: PSy layer in LFRic does not modify the LMA operator objects.
+        Hence, their Fortran intents are always "in" (the data pointed to
```

Comment:
s/the data pointed to by the operators are updated in kernels)/the data updated in the kernels is only *pointed* to from the operator object and is thus not a part of the object)/

Reply:
Thanks for the suggestion, done.